wgsl/index.bs

```diff
@@ -3800,36 +4018,31 @@ When the type is specified, e.g `let foo: i32 = 4`, the initializer expression [
 The value of a <dfn noexport>pipeline-overridable</dfn> constant is fixed at
-pipeline-creation time.
-The value is the one specified by the WebGPU pipeline-creation method, if
-specified, and otherwise is the value of its initializer expression.
-When an [=identifier=] use [=resolves=] to a override-declaration, the identifier denotes that value.
-`override`-declarations [=shader-creation error|must=] meet the following restrictions:
-
-  * The declaration [=shader-creation error|must=] only occur at [=module scope=].
-  * The declaration [=shader-creation error|must=] have at least one of a declared type, an initializer
-      expression, or both.
-  * The declared type, if present, [=shader-creation error|must=] be a [=scalar=].
-  * The initializer expression, if present, [=shader-creation error|must=]:
-      * evaluate to a [=scalar=] type.
-      * evaluate to the declared type if it is present.
-      * be an [=override-expression=].
-  * If the declaration has the [=attribute/id=] applied, the literal operand is
-      known as the <dfn noexport>pipeline constant ID</dfn>, and [=shader-creation error|must=] be an
-      integer value between 0 and 65535.
-  * Pipeline constant IDs [=shader-creation error|must=] be unique within the WGSL program: Two `override`-declarations
-    [=shader-creation error|must not=] use the same pipeline constant ID.
-  * The application can specify its own value for the constant at pipeline-creation time.
-    The pipeline creation API accepts a mapping from overridable constant to a
-    value of the constant's type.
-    The constant is identified by a <dfn export>pipeline-overridable constant identifier string</dfn>,
-    which is the base-10 representation of the [=pipeline constant ID=] if specified, and otherwise
-    the declared [=name=] of the constant.
-  * The <dfn export>pipeline-overridable constant has a default value</dfn> if
-    its declaration has an initializer expression.
-    If it doesn't, it is a [=pipeline-creation error=] if a value is not provided at pipeline-creation time.
-
-Note: Override expressions are a superset of [=const-expressions=].
+[=pipeline creation|pipeline-creation time=].
+The value is one specified by the WebGPU pipeline-creation method, if
+specified, and otherwise is the value of its [=concretization of a
+value|concretized=] initializer expression.
+The [=effective-value-type=] of an override-declaration must be a [=type/concrete=]
+[=scalar=] type.
+
+An initializer expression is optional.
+If present, it must be an [=override-expression=] and represents the <dfn
+noexport>pipeline-overridable constant default value</dfn>.
+If no initializer is specified, it is a [=pipeline-creation error=] if a value
+is not provided at pipeline-creation time.
+
+If the declaration has an [=attribute/id=] attribute applied, the literal
+operand is known as the <dfn noexport>pipeline constant ID</dfn>, and must be a
+unique integer between 0 and 65535.
```

Comment:
Is this an inclusive list, or literally between? (i.e, can I do `@id(0)` and `@id(65535)`) ?